api_session.yaml

```diff
@@ -0,0 +1,95 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Proxy API
+  description: Collection of enpoints to interact with user session.
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+security:
+  - Token: []
+paths:
+  "/session/{fiscalcode}/lock":
```

Comment:
Could we use the pluralized resource name for `sessions`? [Here](https://opensource.zalando.com/restful-api-guidelines/#134) a reference.